build.gradle.kts

```diff
@@ -31,6 +31,20 @@ import de.thetaphi.forbiddenapis.gradle.CheckForbiddenApis
 import de.thetaphi.forbiddenapis.gradle.CheckForbiddenApisExtension
 import org.gradle.api.tasks.testing.logging.TestExceptionFormat
 
+buildscript {
+    dependencies {
+        constraints {
+            // Looks like we cannot access the versions defined in gradle.properties from here
```

Comment:
We can access the properties in `gradle.properties` if we use the built-in functions. For instance the following works:
```
"classpath"("org.ow2.asm:asm:${property("asm.version")}")
```